src/init.c

```diff
@@ -124,6 +125,8 @@ shmem_internal_shutdown(void)
 
     shmem_internal_randr_fini();
 
+    shmem_internal_team_fini();
```

Comment:
IIUC, team_fini needs to occur before transport_fini since team_fini calls transport layer functions to destroy contexts on the teams that it destroys.

Reply:
Yeah, great catch.  It's a good sign that nothing blew up here.  I've been calling `transport_quiet` and `transport_ctx_destroy` _after_ `transport_fini` for quite some time...